montblanc/impl/biro/v2/loaders/loaders.py

```diff
@@ -23,6 +23,10 @@
 import montblanc
 import montblanc.impl.common.loaders
 
+ORDER_CASA = 'casa'
+ORDER_OTHER = 'other'
+VALID_ORDERINGS = [ORDER_CASA, ORDER_OTHER]
```

Comment:
@IanHeywood, @marziarivi  OK, then according to my understanding, Montblanc's default ordering (**time x baseline**, time is row major while baseline is row minor) matches CASA's default ordering.

Marzia, I think then `ORDER_CASA` becomes the default (and current) behaviour, while `ORDER_OTHER` supports something else.

In fact, I think it would be better to rename these to `ORDER_TIME_BL` and `ORDER_BL_TIME`.